.github/pull_request_template.md

```diff
@@ -2,4 +2,12 @@ PR checklist:
 
 - [ ] Documentation is up-to-date
 - [ ] Changelog is up-to-date
+- [ ] Changelog includes:
+  - A reference to a Linear ticket - if no linear ticket is associated, a reference to a GitHub issue number or a semantic name, for example, `gh1234.added` or `logoutput.fixed` is appropriate.
+  - Changlog entry file is named appropriately - for example, `cli-123.added`, `pa-321.fixed`, `gh-1234.changed` or `release-process.infra`
+  - What was changed
+  - Why the change was made
+  - How the change/design changes were made
```

Comment:
```suggestion
- [ ] `changelog.d/<issue>.<type>` is a file with the *what*, *why*, and *how* of the change.
  - \<issue> is `pa-312` (Linear ticket), `gh-1234` (GitHub issue), or `new-gizmo` (unique semantic name)
  - \<type> is `added`, `changed`, `fixed`, or `infra`.
```

Tried to make this more concise, added some more detail, and fixed a typo.

rendered preview:

- [ ] `changelog.d/<issue>.<type>` is a file with the *what*, *why*, and *how* of the change.
  - \<issue> is `pa-312` (Linear ticket), `gh-1234` (GitHub issue), or `new-gizmo` (unique semantic name)
  - \<type> is `added`, `changed`, `fixed`, or `infra`.